--- a/libfprint/drivers/elanmoc2/elanmoc2.c
+++ b/libfprint/drivers/elanmoc2/elanmoc2.c
@@ -1454,6 +1454,7 @@
 static const FpIdEntry elanmoc2_id_table[] = {
   {.vid = ELANMOC2_VEND_ID, .pid = 0x0c00, .driver_data = ELANMOC2_ALL_DEV},
   {.vid = ELANMOC2_VEND_ID, .pid = 0x0c4c, .driver_data = ELANMOC2_ALL_DEV},
+  {.vid = ELANMOC2_VEND_ID, .pid = 0x0c9d, .driver_data = ELANMOC2_DEV_0C5E},
   {.vid = ELANMOC2_VEND_ID, .pid = 0x0c5e, .driver_data = ELANMOC2_DEV_0C5E},
   {.vid = 0, .pid = 0, .driver_data = 0}
 };
